BaristaSwift/Consumer/Consumer/AppDelegate.swift

```diff
@@ -54,7 +55,8 @@ class AppDelegate : UIResponder, UIApplicationDelegate
     
     override init() {
         super.init()
-        
+        SalesforceSwiftLogger.setLogLevel(.debug)
```

Comment:
This should be the default in a `DEBUG` build. Setting this here will enforce the same setting in a `RELEASE` build too.